### ISSUE TITLE
Removing hard coding of JWT_SECRET for seeding docker setup

### ISSUE THREAD

**Issue by bhvngt** [supabase/realtime#411]
# Feature request
Currently JWT_SECRET is hard coded for seeding docker setup

https://github.com/supabase/realtime/blob/569a33c4f3de9fe2c455ec30c651212fe22a3f44/priv/repo/seeds.exs#L15

Can this be paramterised based on env variable just like what is being done for other postgres related arguments in the same file?

## Is your feature request related to a problem? Please describe.

This makes the local setup slightly difficult and also becomes a security issue for any self-hosted setup.

## Describe the solution you'd like

Parameterise JWT_SECRET as env variable just like what is being done for other postgres related arguments in the same file.

**Comment by itisnajim** [comment]
another issue in the same context, 
you can't use tokens signed with a payload contains `"role": "service_role"` (to bypass rls for example)
this will not work:
```js
const client = createClient(
    process.env["SUPABASE_URL"] ?? "",
    process.env["SUPABASE_SERVICE_KEY"] ?? "", // but for SUPABASE_ANON_KEY no errors occur
);
```

**Comment by w3b6x9** [comment]
>Parameterise JWT_SECRET as env variable just like what is being done for other postgres related arguments in the same file.

@bhvngt yep, will update the image to do this.

**Comment by w3b6x9** [comment]
`JWT_SECRET` now defaults to `API_JWT_SECRET` and is available in `supabase/realtime:v2.1.0`.
